functions/Backup-DbaDatabase.ps1

```diff
@@ -453,8 +453,17 @@ function Backup-DbaDatabase {
 
             if ($CompressBackup) {
                 if ($db.EncryptionEnabled) {
-                    Write-Message -Level Warning -Message "$dbName is enabled for encryption, will not compress"
-                    $backup.CompressionOption = 2
+                    $minVerForTDECompression = [version]'13.0.4446.0' # SQL Server 2016 CU 4
+                    $flagTDESQLVersion = $minVerForTDECompression -le $Server.version
+                    $flagTestBoundMaxTransferSize = test-bound 'MaxTransferSize'
+                    $flagVcorrectMaxTransferSize = $flagTestBoundMaxTransferSize -and ($MaxTransferSize -gt 64kb)
```

Comment:
What does the "V" in "flagVcorrectMaxTransferSize" stand for?

Reply:
It was a typo.